### ISSUE TITLE
Pre-pull Request to fix onion

### ISSUE THREAD

**Issue by IronBlood** [TechEmpower/FrameworkBenchmarks#1592]
Hi, there

I made some changes in mine branch to make a successful build on travis, but I'm not sure whether these commits are correct. Could someone review these commits?

## sudo required in IronBlood/FrameworkBenchmarks@c4961b47bde14a98f6df1fb6efbfc7af3aadfd55
[The previous build](https://travis-ci.org/IronBlood/FrameworkBenchmarks/builds/60491207) stopped at 
```
sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
```
So that I added `sudo: required` to `.travis.yml` file.

## compile task changes for `frameworks/C/onion/Makefile` in IronBlood/FrameworkBenchmarks@8d71ab034073f23c81d8378ca998237bc2a3b2d8 and IronBlood/FrameworkBenchmarks@f776b67cd37a3e98b68c7934e053468cf36e7e76

It seem that onion should be compiled together, instead of compiling `onion/src` and `onion/otemplate` seperately, so I made some tricks, shown as:

```
 libonion_static.a: onion
-	(cd onion && mkdir -p build && cd build && cmake .. && cd src && make -j4)
+	(cd onion && mkdir -p build && cd build && cmake .. -DCMAKE_C_FLAGS="-Werror" -DCMAKE_CXX_FLAGS="-Werror" && make VERBOSE=1)
 	cp onion/build/src/onion/libonion_static.a .

 otemplate:
-	(cd onion/build/tools/otemplate/; make)
+	(cd onion/build/tools/otemplate/)
```

And this time, [the build](https://travis-ci.org/IronBlood/FrameworkBenchmarks/builds/60502430) was passed. So here's my question:
- Is `sudo: required` necessary in `.travis.yml`?
- What does the verification summary mean? Does it mean all the tests pass as expected?

Regards.

**Comment by msmith-techempower** [comment]
@IronBlood Thanks very much for this - I tested it locally and it seems to be working perfectly, so I have included it in my [latest branch](https://github.com/TechEmpower/FrameworkBenchmarks/compare/4e0f564ddf80...ab3b3f618684#diff-7b4a8c7cc2e549804a4384b5171512e8R13) that will hopefully be getting merged in soon.

**Comment by IronBlood** [comment]
Great to hear that :smile:

**Comment by hamiltont** [comment]
> Is sudo: required necessary in .travis.yml?

Shouldn't be, it's currently the default, but it doesn't hurt to make it explicit. [some context if you're intereseted](http://docs.travis-ci.com/user/workers/container-based-infrastructure/)

> What does the verification summary mean? Does it mean all the tests pass as expected?

Yup! We've verified that your responses are what we expect to see for those tests, that you've sent the correct headers back, and even checked to ensure you're not making any common mistakes like including unnecessary quoting. You're all good, no issues found! Thanks

**Comment by IronBlood** [comment]
Thanks, @hamiltont.

As @msmith-techempower has patched these codes in ab3b3f6186842ef2ad0eb65d33eb73c3ef3159e4, I think it's time to close this issue.

**Comment by hamiltont** [comment]
:+1:
